docs/_documentations/remotedeploy-eclipse.md

```diff
@@ -45,52 +47,48 @@ Then complete the docker registryconnection details:
 
 ![Adding new remote project](./images/remoteeclipse/newProject.png)
 
-Select the template type, project name and your new project should be built and after a few seconds begin running in the cloud.
+2\. Select the template type, and name your project. Your new project is built and after a few seconds begins running in the cloud.
 
 ![Adding new remote project](./images/remoteeclipse/runningProject.png)
 
 ## Copying an existing local project to the cloud
 
-Copying an existing local project "myFirstNodeProject" over to the remote Codewind deployment.
-
-To begin :
+The folowing procedure copies an existing local project `myFirstNodeProject` over to the remote Codewind deployment:
 
-Right-click the remote deployment and click the "Add existing project button"
+1\. Right-click the remote deployment and click the `Add existing project button`:
 
 ![Add existing project](./images/remoteeclipse/addExistingProject.png)
 
-Navigate to the folder containing your local project files, or choose the previously created "myFirstNodeProject" from the workspace. Click next to proceed :
+2\. Navigate to the folder containing your local project files, or choose the previously created `myFirstNodeProject` from the workspace. Click `Next` to proceed:
 
 ![Add to Codewind](./images/remoteeclipse/existingProject.png)
 
-Codewind will ask you to confirm the project type identified as NodeJS:
+3\. Codewind prompts you to confirm the project type identified as NodeJS:
 
 ![Confirm Project Type](./images/remoteeclipse/confirmProjectType.png)
 
-click Next
+4\. Click `Next`.
 
-The project files are copied over to the Codewind server and the new "myFirstNodeProject" appears in the Codewind panel. Codewind will begin building your application code and Docker image, moments later the Project Image will be uploaded to DockerHub and used by your cloud deployment to provision a new pod:
+The project files are copied over to the Codewind server and the new `myFirstNodeProject` appears in the Codewind panel. Codewind builds your application code and Docker image, and moments later, the project image is uploaded to DockerHub and used by your cloud deployment to provision a new pod:
 
 ![Build Success](./images/remoteeclipse/buildSuccess.png)
 
 `myFirstNodeProject` on `MyIBMCloud` is now running and ready.
 
-Congratulations, In this topic you:
-
-1. Deployed a new Codewind install into Openshift
-2. Configured your IDE to use this new deployment
-3. Registered all the necessary security parameters
-4. Created a new project built that builds and runs in the cloud
-5. Copied an existing local project to build and run in the cloud
+Congratulations! In this topic you:
 
-In the next step you will learn how to remove an existing Codewind deployment
+1. Deployed a new Codewind install into Openshift.
+2. Configured your IDE to use this new deployment.
+3. Registered all the necessary security parameters.
+4. Created a new project built that builds and runs in the cloud.
```

Comment:
Typo? Should "built" be "build"?

Reply:
Should be `4. Created a new project that builds and runs in the cloud.` corrected.